docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+
+2. Click the **objectfilter_cam** panel to open your detector camera controls.
+
+3. Toggle **View objectfilter_cam** to the "on" position.
+   This displays a live feed from your webcam with detection bounding boxes overlaid on it.
+
+4. The detector can recognize humans with and without hard hats on.
+   Try positioning yourself in front of the camera without anything on your head, and then with a helmet on, and watch the bounding boxes appear.
+
+   {{<imgproc src="/tutorials/helmet/no-hard-hat1.png" resize="x1100" declaredimensions=true alt="A person with no hard hat on, with a bounding box labeled No-Hardhat around her head." >}}
+
+## Configure data capture and sync
+
+Viam's built-in [data management service](/data/) allows you to, among other things, capture images and sync them to the cloud.
+Configure data capture on the `objectfilter` camera to capture images of people without hard hats:
+
+1. First, you need to add the data service to your machine to make it available to capture data on your camera.
+
+   Navigate to the **Services** subtab of your machine's **Config** tab.
+
+   Click **Create service**.
+   Type "data" and click **data management / RDK**.
+   Name your data management service `data-manager` and click **Create**.
+
+   Leave all the default data service attributes as they are and click **Save config**.
+
+2. Now you're ready to enable data capture on your detector camera.
+   Navigate to the **Components** subtab.
+   Locate the `objectfilter-cam` panel.
+
+3. Click **Add method**.
+   Click the **Type** dropdown and select **ReadImage**.
+   Set the capture frequency to `0.2` images per second (equivalent to one image every 5 seconds).
+   You can always change the frequency to suit your use case.
+
+   {{<imgproc src="/tutorials/helmet/datacapture-config.png" resize="x1100" declaredimensions=true alt="The objectfilter cam configured with data capture set to read images at 0.2 hertz. Mime type is left at the default image/jpeg." >}}
+
+### Test data capture and sync
+
+To make sure the detector camera is capturing and syncing labeled images:
+
+1. Position yourself in front of your webcam for 30 seconds or so to let it capture a few images of a person without a hard hat on.
+
+2. Navigate to your [**DATA** page](https://app.viam.com/data/view?view=images) in the Viam app.
+   You should see some images with bounding boxes on them.
+   If you do not, try refreshing the page.
+
+   {{<imgproc src="/tutorials/helmet/synced-data.png" resize="x1000" declaredimensions=true alt="The data manager page in the Viam app, displaying three images of a person with a bounding box labeled NO-Hardhat around her face." >}}
+
+3. You can also try this with a hard hat on your head.
+
+   {{<imgproc src="/tutorials/helmet/hardhat-data.png" resize="x1000" declaredimensions=true alt="The data manager page in the Viam app, displaying three images of a person with a bounding box labeled NO-Hardhat around her face." >}}
```

Comment:
see above - want to check you're using images without copyright

Reply:
Yep done!